app/src/main/res/layout-v21/appview_poa_layout.xml

```diff
@@ -1,130 +1,124 @@
 <?xml version="1.0" encoding="utf-8"?>
 <androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
-    xmlns:tools="http://schemas.android.com/tools"
+  xmlns:app="http://schemas.android.com/apk/res-auto"
```

Comment:
What happened in this file? The only real change I see here is the icon name. Is it because my code style is different from yours on xml files due to the update to studio?